src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Enum names should be in a singular form.

```suggestion
public enum Color {
```